README.md

```diff
@@ -33,10 +33,13 @@ We currently support a few LLM models targeting text generation scenarios:
 
 `pip install optimum-tpu -f https://storage.googleapis.com/libtpu-releases/index.html`
 
+`export PJRT_DEVICE=TPU`
+
+
 ## Inference
 
 `optimum-tpu` provides a set of dedicated tools and integrations in order to leverage Cloud TPUs for inference, especially
-on the latest TPU version `v5e`. 
+on the latest TPU version `v5`. 
```

Comment:
it is actually called `v5e`, see https://cloud.google.com/tpu/docs/v5e

Reply:
There is also v5p that's why i wanted to say v5 directly, but with #108 this sentence is actually removed to make it more general

Reply:
In theory we could support v5p, but we haven't done any testing on that platform yet. You can remove the sentence as done on #108 and they will rebase later.